docs/data/stashcache/install-origin.md

```diff
@@ -11,87 +11,89 @@ to export its data to the StashCache data federation.
---------------------------------
+Installing the origin
+---------------------
 
-The StashCache daemon consists of an XRootD server and an HTCondor-based service for collecting and reporting
-monitoring data about the origin. To simplify installation, OSG provides convenience RPMs that install all required
+The origin service consists of an XRootD daemons and its dependencies for the authentication infrastructure.
+To simplify installation, OSG provides convenience RPMs that install all required
 software with a single command:
 
 ```console
-root@host # yum install stashcache-origin-server
+root@host # yum install --enablerepo=osg-development stash-origin
 ```
 
+!!! note
+    This document covers a completely overhauled stash origin service that depends on XRootD 4.9.0; while this
+    has not yet been released into the OSG production repository, we strongly recommend using this version.
+
 For this installation guide, we assume that the data to be exported to the federation is mounted at `/stash`
 and owned by the `xrootd:xrootd` user.
 
 Configuring the Origin Server
 -----------------------------
 
-The `stashcache-daemon` package provides a default configuration file,
-`/etc/xrootd/xrootd-stashcache-origin-server.cfg`, which must be customized for your origin.
+The `stash-origin` package provides a default configuration files in
+`/etc/xrootd/xrootd-stash-origin.cfg` and `/etc/xrootd/config.d`.  Administrators may provide additional configuration
+by placing files in `/etc/xrootd/config.d` of the form `/etc/xrootd/config.d/1*.cfg` (for directives that need to
+be processed BEFORE the OSG configuration) or `/etc/xrootd/config.d/9*.cfg` (for directives that are processed
+AFTER the OSG configuration).
+
+All _mandatory_ configuration that the admin must make is in `/etc/xrootd/config.d/10-origin-site-local.cfg`; EVERY
+directive there must be configured.
 
-The most common lines to customize are:
+The three mandatory directives that must be customized are:
 
-* `oss.localroot /stash`: Change the `localroot` to the location where your data is mounted on
-  the origin server; default is `/stash`.
-* `all.export /<YOUR VO>`: A sub-directory within the `localroot` directory that will be exported.
-  If multiple directories must be exported, you may specify `all.export` multiple times.
+* `origindir`: the mounted filesystem path to export.  This document refers to this as `/mnt/stash`.
+* `originexport`: the file within the `origindir` to export.  This MUST be in your supported VO's registered namespace.
+* `sitename`: the resource name registered with OSG.
 
-For example, if the HCC VO would like to set up an origin server exporting from the mountpoint `/mnt/bigdata`,
-but only export the subdirectories `/mnt/bigdata/hcc/bio/datasets` and `/mnt/bigdata/hcc/hep/generators`,
-they would use the following configuration:
+For example, if the HCC VO would like to set up an origin server exporting from the mountpoint `/mnt/stash`,
```

Comment:
```suggestion
For example, if the HCC VO would like to set up an origin server exporting from the mountpoint `/stash`,
```

Reply:
Yeah, that's what I did originally -- however, I'd like to keep `/mnt` to emphasize that this is on the local filesystem.  I don't think most admins would mount their actual filesystem inside `/`.

Reply:
If you want to keep `/mnt`, then we need to change it everywhere else in the document and the default values for `cachedir` and `origindir` in the `*-site-local.cfg` files in xcache.

Reply:
Not that I'm not willing to do that -- we haven't tagged xcache 1.0.1 and if you think /mnt/stash is a better value then we can use it, but it's nice to have documents consistent with the config... and uh they should be consistent with themselves, too :)

Reply:
Good point.  Let's change the default suggested `cachedir` and `origindir`.

(Is there somewhere else in this document I missed?)

Reply:
Just at the top. I made a suggestion.